src/aznfswatchdog

```diff
@@ -53,10 +47,11 @@ next_ip_change_detection_epoch=0
 #
 next_daily_run_epoch=$(expr $(date +%s) + 60)
 
+AZNFS_VERSION=3
+
 # Load common aznfs helpers.
 . /opt/microsoft/aznfs/common.sh
```

Comment:
nit, let's set AZNFS_VERSION just before the sourcing of common.sh, for more clarity.

```
# Load common aznfs helpers.
AZNFS_VERSION=3
. /opt/microsoft/aznfs/common.sh
```

Reply:
@linuxsmiths isn't this a bit confusing calling it AZNFS_VERSION? I would prefer NFS_VERSION. What do you say?